libraries/AP_Scripting/examples/servo_override.lua

```diff
@@ -0,0 +1,21 @@
+-- Testing set_output_pwm_chan_counter_zero and set_output_pwm_chan_counter
+--
+-- This will MAIN1 servo to max pwm for 1 second (50 counts),
```

Comment:
Is this comment correct? 
If I'm reading the script correctly, the update interval is 2000 milliseconds
What is "50 counts"?

Reply:
That's left over from the initial version of the method. I'll update that to be 1000 ms at each position and 1000 ms of normal operation between. The end behavior is this same. Or did I push the 5000 ms override I was messing around with on my desk.

Reply:
Changes made and pushed.